workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectCreation/GameObjectCreatorFromMetadata.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections.Generic;
+using System.IO;
+using Generated.Improbable;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using UnityEngine;
+using Transform = Generated.Improbable.Transform.Transform;
```

Comment:
this is from the TransformSynchronization module. We should not depend on another feature module  for this